Readme.md

```diff
@@ -3,16 +3,65 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T873926)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# Grid for Blazor - How to use DxUpload component to upload images to the grid
+
+This example demonstrates how to use [DxUpload](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxUpload) component inside the grid edit form template to upload images to the server and then display them in the grid.
+
+![Edit form with DxUpload component](grid-edit-form.png)
+
+In this example, the data source contains paths to images that are maintained on the server. [DxGrid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) displays the images in the cell template ([CellDisplayTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridDataColumn.CellDisplayTemplate)).
+
+```razor
+<DxGridDataColumn FieldName=@nameof(TestModel.ImageUrl) Caption="Image">
+    <CellDisplayTemplate>
+        <img style="max-width:300px" src="@context.Value" />
+    </CellDisplayTemplate>
+</DxGridDataColumn>
+```
+
+The grid's edit form template ([EditFormTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.EditFormTemplate)) contains [DxUpload](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxUpload) component that allows you to upload images to the server. 

```

Comment:
Earlier you refer to it as `the grid edit form template`  but here it is `the grid's edit form template`?

Reply:
```suggestion
The grid edit form template ([EditFormTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.EditFormTemplate)) contains [DxUpload](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxUpload) component that allows you to upload images to the server. 

```